README.md

```diff
@@ -107,6 +108,50 @@ env:
   - DEPLOY_WORKER=true
 ```
 
+## Releasing
+
+1. Checkout a new release branch from `develop`.
+2. Bump the version of each of the subpackages (frontend, backend, worker).
+3. Stage changes for subpackage `package.json` and `package-lock.json`.
+4. Bump the version of the main package.
+   - You will need to use `--force` as the working directory is not clean.
+5. Push both the version commit and tag to GitHub.
+6. Open a new pull request and consolidate changelist from the commits made since the previous release.
+7. Merge pull request into master to trigger Travis build and deploy.
```

Comment:
I'll modify this to explicitly state that we don't squash into master

Reply:
```suggestion
7. Merge pull request into master to trigger Travis build and deploy. Note that we do not squash commits when merging into master.
```
How about this?

Reply:
Yup looks good